plugins/bulk_upload/base/services/BulkService.php

```diff
@@ -445,21 +445,72 @@ function abortAction($id)
 	 */
 	public function syncGroupUsersAction($userId, $groupIds, $removeFromExistingGroups = true, $createNewGroups = true)
 	{
-		$kuser = kuserPeer::getKuserByPartnerAndUid($this->getPartnerId(), $userId);
-		if (!$kuser || $kuser->getType() != KuserType::USER)
+		$groupLimit = kConf::get('user_groups_sync_threshold','local',50);
+		$bulkUpload = null;
+
+		$kUser = kuserPeer::getKuserByPartnerAndUid($this->getPartnerId(), $userId);
+		if (!$kUser || $kUser->getType() != KuserType::USER)
+		{
 			throw new KalturaAPIException(KalturaErrors::INVALID_USER_ID, $userId);
+		}
 
-		if(!$groupIds)
+		if (!$groupIds)
```

Comment:
No need to validate that value exist since this parameter does not have default value, user cannot call this action without sending it.